tpot/tpot.py

```diff
@@ -113,7 +113,8 @@ def __init__(self, population_size=100, generations=100,
         self.pset.addPrimitive(self.gradient_boosting, [pd.DataFrame, float, int, int], pd.DataFrame)
         self.pset.addPrimitive(self._combine_dfs, [pd.DataFrame, pd.DataFrame], pd.DataFrame)
         self.pset.addPrimitive(self._variance_threshold, [pd.DataFrame, float], pd.DataFrame)
-        self.pset.addPrimitive(self._select_kbest, [pd.DataFrame, int], pd.DataFrame) 
+        self.pset.addPrimitive(self._select_kbest, [pd.DataFrame, int], pd.DataFrame)
+        self.pset.addPrimitive(self._select_fwe, [pd.DataFrame, int], pd.DataFrame) # added selectfwe
```

Comment:
No need to have a comment saying `# added selectfwe` -- git and GitHub track those changes for us. :-)